diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/PackageNaming.kt

```diff
@@ -36,8 +37,8 @@ import java.util.concurrent.atomic.AtomicInteger
 @Suppress("ForbiddenComment")
 class PackageNaming(private val configRules: List<RulesConfig>) : Rule("package-naming") {
     private var isFixMode: Boolean = false
+    private var domainName: String? = null
```

Comment:
You could make it non-nullable lateinit and assign only if it's present. Now you have a lot of `!!` and it should be avoided,